vodd01-sp2

Dandicek opened the pull request:
Link to my application: https://eso.vse.cz/~vodd01/sp2/

Dandicek committed:
vodd01-sp2

Link to my application: https://eso.vse.cz/~vodd01/sp2/

nvbach91 reviewed (changes requested):
prace neni odevzdana

Dandicek committed:
Add files via upload

Dobrý večer,
omlouvám se za pozdní odevzdání, musel jsem pro svou API vytvořit proxyserver, aby se mi zobrazovaly výsledky.

Dandicek committed:
Add files via upload

Dandicek commented:
Dobrý večer,
omlouvám se za pozdní odevzdání, soubory by měly být nahrané. Mohu před zítřejší obhajobou dále upravovat?

Dandicek committed:
Add files via upload

Upravená verze na obhajobě 31.1.

nvbach91 committed:
Delete www/proxyserver.js

nvbach91 commented:
Ještě musíte opravit SP1, máte tam málo bodů viz https://github.com/nvbach91/4IZ268-2024-2025-ZS/issues/3

nvbach91 commented:
@Dandicek

Dandicek commented:
> Ještě musíte opravit SP1, máte tam málo bodů viz #3

Dobrý den,

už jsem opravil i první semestrální práci.

nvbach91 commented:
Odevzdani opravy viz pokyny https://github.com/nvbach91/4IZ268-2024-2025-ZS/wiki/P-Pokyny-SP1

Dandicek commented:
> Odevzdani opravy viz pokyny https://github.com/nvbach91/4IZ268-2024-2025-ZS/wiki/P-Pokyny-SP1

Vytvořil jsem nový Issue pro opravu a popsal opravené chyby.
